### DISCUSSION TITLE
Need information about the GitHub plan

### DISCUSSION THREAD

**Discussion by RohanGhatkamble** [community/community#42642] [Enterprise]
### Select Topic Area

Product Feedback

### Body

We need information about the GitHub plan, we have purchased "GitHub Team" plan, but right now it is showing "GitHub Enterprise" plan since April 2022. Can we downgrade it?

**Comment by radar-blip** [comment]
For something like that you would have to contact [Support](https://support.github.com/contact/account)

**Reply by RohanGhatkamble** [reply]
I already raised case 15 days back, but nobody replying on that.

**Reply by DSDmark** [reply]
Hi @RohanGhatkamble,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Reply by DSDmark** [reply]
If you want to downgrade from GitHub Enterprise to GitHub Team, you will need to cancel your GitHub Enterprise subscription and then purchase a new GitHub Team subscription. You can do this through the billing settings in your organization's settings on GitHub.
